pkg/storage/constants/constants.go

```diff
@@ -0,0 +1,21 @@
+package constants
```

Comment:
does this need its own package?
why not under pkg/storage/constants.go itself under storage pkg?

Reply:
Moved into storage package

Reply:
And I had to move them back due to the new changes.

Reply:
pkg/storage/constants.go

so "import pkg/storage" is sufficient?

Reply:
I also needed to move the cache driver interface to the cache package, as it would have created some more cyclic imports on the dynamodb implementation (in the cache package) if left in storage